www/docs/en/dev/guide/platforms/android/lifecycle.md

```diff
@@ -116,10 +196,21 @@ onResume(event) {
     if(event.pendingResult) {
         // Figure out whether or not the plugin call was successful
         if(event.pendingResult.pluginStatus === "OK") {
-            successCallback(event.result);
+            successCallback(event.pendingResult.result);

```

Comment:
What does this successCallback refer to? Perhaps you want to give a realistic example with cordova-plugin-camera. I would recommend including the same example in cordova-pluign-camera docs.